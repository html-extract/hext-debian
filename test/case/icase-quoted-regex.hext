<a class=~"^[drow]+$"i class:class />
