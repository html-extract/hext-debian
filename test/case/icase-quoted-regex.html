<html>
  <head>
  </head>
  <body>
    <a class="WORD"></a>
    <a class="word"></a>
    <a class="WoRd"></a>
    <a class="Word"></a>
    <a class="NOPE"></a>
    <a class="nOpE"></a>
    <a class="nope"></a>
    <a class="no"></a>
  </body>
</html>
