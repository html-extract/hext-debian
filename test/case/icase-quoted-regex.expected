{"class":"word"}
{"class":"Word"}
{"class":"WoRd"}
{"class":"WORD"}
